### PR TITLE
Add 'set backupcopy=yes' to vimrc.j2

### PR DESCRIPTION
**Pull request by altesockensuppe**
##### ISSUE TYPE

 - Feature Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by sliverc — APPROVED** [review]
LGTM

**Comment by fujexo** [comment]
I'm not sure if this is good to deploy on every machine?
Can we make this configurable via variable? 

Or is the backup copy removed after saving?

File: templates/etc/vimrc.j2 [hunk 1]
```diff
@@ -75,3 +75,6 @@ set suffixes=.bak,~,.swp,.o,.info,.aux,.log,.dvi,.bbl,.blg,.brf,.cb,.ind,.idx,.i
 
 " Enable modelines if there are some in files.
 set modeline
+
+" make a copy of the file and overwrite the original one
+set backupcopy=yes
```
Comment (fujexo) [inline templates/etc/vimrc.j2:81]: Please make `backupcopy` configurable using a variable. The default should be without `backupcopy` set to `yes`.
We can then create a group variable for all docker hosts which enables the `backupcopy` on them.

**Review by fujexo — CHANGES_REQUESTED** [review]

**Comment by fujexo** [comment]
LGTM

**Review by fujexo — APPROVED** [review]

**Pull request merged**